### PR TITLE
Introduce configurable target frame rate (defaults to infinite)

### PR DESCRIPTION
**Pull request by abesto**
As discussed in #42. Compared to #42, this PR

 * adds extra logic to have a "target framerate" as opposed to a "max framerate"
 * makes the target framerate configurable

### CODE REVIEW

File: src/layouts/ltrTreeLayout/index.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ class LTRTreeLayout {
   run (graph, dimensions, layoutComplete) {
     const layoutWorker = LayoutWorker();
     const layoutWorkerComplete = (event) => {
-      graph.nodes.forEach(node => {
+      graph.nodes.forEach((node) => {
```
Comment (abesto) [inline src/layouts/ltrTreeLayout/index.js]: No idea what added this; maybe Emacs, maybe the linter. Let me know if I should remove it.

**Comment by abesto** [comment]
Thank you!

**Comment by jolexa** [comment]
@abesto Hi there. Do you have an example of actually configuring it?

**Comment by abesto** [comment]
@jolexa only here: https://github.com/Netflix/vizceral-react/commit/243575b5b5164bdc2b8ceee9b381a8136a10577a

**Pull request merged**